### ISSUE TITLE
quit crash auto

### ISSUE THREAD

**Issue by xueqing0622** [nunchaku-ai/ComfyUI-nunchaku#778]
nunchaku-1.2.0+torch2.9-cp311-cp311-win_amd64.whl 
pytorch version: 2.9.1+cu130
Python version: 3.11.9
ComfyUI version: 0.8.2
ComfyUI frontend version: 1.36.13

got prompt
Using pytorch attention in VAE
Using pytorch attention in VAE
VAE load device: cuda:0, offload device: cpu, dtype: torch.bfloat16
[FlowMatch Scheduler] Auto-detected device: CUDA
CLIP/text encoder model load device: cuda:0, offload device: cpu, current: cpu, dtype: torch.float16
Requested to load ZImageTEModel_
loaded completely; 27960.80 MB usable, 7672.25 MB loaded, full load: True
Z-Image Batch Encode Output:
  Batch size: 8
  Condition shape: torch.Size([8, 77, 2560])
  Pooled shape: torch.Size([8, 2560])
unet_dtype: torch.bfloat16, manual_cast_dtype: None, svdq_linear_dtype: torch.bfloat16
model weight dtype torch.bfloat16, manual cast: None
model_type FLOW
Requested to load Lumina2
loaded completely; 23819.71 MB usable, 4007.28 MB loaded, full load: True
  0%|                                                                                            | 0/1 [00:00<?, ?it/s]Assertion failed: rotary_emb.shape[0] * rotary_emb.shape[1] == M, file C:\Users\muyangl\actions-runner\_work\nunchaku\nunchaku\src\kernels\zgemm\gemm_w4a4_launch_impl.cuh, line 353
请按任意键继续. . .

**Comment by svenhimmelvarg** [comment]
I've just grappled crash of comfyui on model load  for the last 30 minutes. Had to roll back to v1.1.0

**Comment by LiJT** [comment]
把 fp16 fast accumalation 去掉才可以运行。等修复吧。。。。。。我也是恢复 1.1 才能正常运行

**Comment by devgdovg** [comment]
Please check if your problem is the same as described in this PR. https://github.com/nunchaku-ai/ComfyUI-nunchaku/pull/781
If it is, removing `--fast fp16_accumulation` argument when starting ComfyUI will avoid the problem temporarily. And you may update ComfyUI-nunchaku after the PR is merged and new version is released.

**Comment by ssitu** [comment]
It happens with batch size > 1 for me. It seems to have started after c43921f00ba0e09fd920e9baaac229de81e694b3

```
pytorch version: 2.10.0+cu130
Set vram state to: LOW_VRAM
Using pytorch attention
Python version: 3.13.2 (tags/v3.13.2:4f8bb39, Feb  4 2025, 15:23:48) [MSC v.1942 64 bit (AMD64)]
ComfyUI version: 0.11.1
Nunchaku version: 1.2.1
ComfyUI-nunchaku version: 1.2.1
platform is win32
...
loaded completely; 5706.80 MB usable, 4335.41 MB loaded, full load: True
  0%|                                                                                            | 0/8 [00:00<?, ?it/s]
D:\GitHub\ComfyUI\ComfyUI\.venv\Lib\site-packages\nunchaku\utils.py:47: UserWarning: Using a non-tuple sequence for multidimensional indexing is deprecated and will be changed in pytorch 2.9; use x[tuple(seq)] instead of x[seq]. In pytorch 2.9 this will be interpreted as tensor index, x[torch.tensor(seq)], which will result either in an error or a different result (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\pytorch\torch\csrc\autograd\python_variable_indexing.cpp:353.)
  result[[slice(0, extent) for extent in tensor.shape]] = tensor
cache miss and created, cache_key: (13215415296, torch.Size([1, 224, 1, 64, 2, 2])), orig shape: torch.Size([1, 224, 1, 64, 2, 2]), packed shape: torch.Size([1, 256, 128])
x shape: torch.Size([2, 224, 3840]), freqs_cis shape: torch.Size([1, 256, 128])
Assertion failed: rotary_emb.shape[0] * rotary_emb.shape[1] == M, file C:\Users\nunchaku\Desktop\actions-runner\_work\nunchaku\nunchaku\src\kernels\zgemm\gemm_w4a4_launch_impl.cuh, line 353
```
